astropy/tests/tests/test_run_tests.py

```diff
@@ -15,7 +15,3 @@ def test_module_not_found():
 def test_pastebin_keyword():
     with pytest.raises(ValueError):
         run_tests(pastebin="not_an_option")
-
-
-def test_unicode_literal_conversion():
```

Comment:
I think this is a leftover from Python 2 days. Now this test makes no sense to me. Definitely unrelated to test runner.

Reply:
agreed